apis/v1alpha1/httproute_types.go

```diff
@@ -19,18 +19,34 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// +genclient
```

Comment:
According to the kubebuilder migration [guide](https://book.kubebuilder.io/migration/guide.html), `genclient` is no longer needed.

Reply:
I tried to do this and it resulted in a huge diff. I think it broke our clientgen. This may need a follow up issue.

Reply:
OK, let's leave it as-is and I created https://github.com/kubernetes-sigs/service-apis/issues/361 to track the issue.